src/DatabaseSeeker.php

```diff
@@ -74,7 +74,7 @@ public function search(Builder $builder, int $page = 1, int $pageSize = null): S
      *
      * @param string[] $keywords
      */
-    private function performSearch(Builder $builder, array $keywords, int $page, ?int $limit): SearchResult
+    private function performSearch(Builder $builder, array $keywords, int $page, ?int $limit = null): SearchResult
```

Comment:
That's not really a necessary change since the method isn't called anywhere without this parameter, but since it's privat, why not.